server/pxf-jdbc/src/main/java/org/greenplum/pxf/plugins/jdbc/JdbcBasePlugin.java

```diff
@@ -479,7 +479,7 @@ private Connection getConnectionInternal() throws Exception {
      * @param connection connection to close
      * @throws SQLException throws when a SQLException occurs
      */
-    private static void closeConnection(Connection connection) throws SQLException {
+    static void closeConnection(Connection connection) throws SQLException {

```

Comment:
```suggestion
    protected static void closeConnection(Connection connection) throws SQLException {

```
to allow it to be called by subclasses only

Reply:
Changed class modifier